### ISSUE TITLE
#### Prepare conda installation.

### ISSUE THREAD

**Issue by pgierz** [FESOM/pyfesom2#156]
Hope to fix conda installation on the way, currently it seems to be totally broken.

Paul: I can have a look at this, would you be opposed to modifying the CI so that it makes automatic conda publication on any tag push? I had recently been playing around with that for one of my private projects.

**Comment by koldunovn** [comment]
I would love that! Let me know what you need from my side - I think we should make access for you (or CI) to https://github.com/conda-forge/pyfesom2-feedstock ?

**Comment by pgierz** [comment]
Maybe we at some point need a robot account ;-) If I could be admin on the feedstock, that should give me everything I need. I might need to get in touch with you if I get stuck, my "publishing" has so far only been with PyPI, so I still need to learn how conda does things. There are already github action templates set up though, so I would as a first attempt just follow those.

**Comment by koldunovn** [comment]
So then I will not do the release just now, so that you can play around with automatic publishing, and try to merge changes (if there will be any) to `daskification` branch?

I added you as a maintainer: https://github.com/conda-forge/pyfesom2-feedstock/issues/3
Here are the instructions on how to update the package: https://conda-forge.org/docs/maintainer/updating_pkgs.html

Most important at this stage would be to 
 - [ ] remove Basemap, that still hangs there for some stupid reason,
 - [ ] add new dependencies,
 - [ ] make sure the MANIFEST is working (now the geojson folder is not copied during the installation)

**Comment by pgierz** [comment]
New dependencies can be detected and added automatically, see #157

**Comment by koldunovn** [comment]
@pgierz can we close this?

**Comment by pgierz** [comment]
Yes. We can reactivate this if demand rises, but auto-publishing is at the moment just a pipe dream which would be "nice to have". Other things are more important at the moment though
